### DISCUSSION TITLE
How to synchronize agents immediately

### DISCUSSION THREAD

**Discussion by Rohrreiniger** [amidaware/tacticalrmm#1722] [Q&A] [answered]
Hello and happy new year!

Foremost I would like to thank you for the Trmm! It has already helped me a lot to get my Windows clients on the same Windows patch level for the first time. Thanks for that and more!

Here is my issue: I have created an automation job that is supposed to shut down certain clients at a certain time using a command (shutdown.exe /F /S /T 2). Unfortunately, I can no longer get it under control. In other words, I had disabled the automation job but the clients are still shutting down. I have now set an expiry date and removed/disabled the task. 

Is there a way to synchronize these new settings ad hoc to the agents? TIA!

Screenshot:
![image](https://github.com/amidaware/tacticalrmm/assets/58429643/3e1ddcd9-f8f3-4b83-9848-46099fcc0f1c)

**Comment by Rohrreiniger** [comment]
I am not sure if this is the same topic, but I have the following going on for a couple of days without changing/updating the agents:

![grafik](https://github.com/amidaware/tacticalrmm/assets/58429643/855a8ece-8994-48c0-9edc-6b73d7b3097b)

Two agents appear to have been updated (completed pending actions) but the icon is still displayed.

**Comment by silversword411** [comment, marked as answer]
Understand what gets updated when
https://docs.tacticalrmm.com/tipsntricks/#understanding-refreshing-trmm-data

Have you refreshed?

Agent checkin is every 120 seconds, you don't have to wait long

**Reply by Rohrreiniger** [reply]
Thanks. Yes, I refreshed.
